### PR TITLE
Env

### PR DESCRIPTION
**Pull request by aaaronhsu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,4 +35,6 @@ jspm_packages
```
Comment (ChristianOkokhere) [inline .gitignore:35]: nit delete space

**Review by ChristianOkokhere — CHANGES_REQUESTED** [review]
fix nit before merge

**Review by aaaronhsu — COMMENTED** [review]
it is perfect the way it is

**Review by ChristianOkokhere — APPROVED** [review]
this is better, we must stay vigilant with respects to our formatting

**Pull request merged**